backend/src/main.ts

```diff
@@ -3,6 +3,7 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  app.enableCors();
```

Comment:
Should make another ticket/PR for the backend changes.

Reply:
Yeah, the scope was to setup up the website. I would say making another ticket and PR would make it more organized and cleaner.

Reply:
I totally see your point. What I was thinking was that not making a separate ticket/PR might help make PR's history cleaner, especially when it looked like it is part of setting up the website. Maybe I don't have a good understanding of best practices here, do you have any resources you can share that might help me understand this subject better?